### PR TITLE
Upgraded to PHP CS Fixer v3

### PR DESCRIPTION
**Pull request by blankse**
See other pimcore repositories

### CODE REVIEW

File: .github/workflows/php-cs-fixer.yaml [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
         steps:
             - uses: actions/checkout@v2
               with:
                   ref: ${{ github.head_ref }}
 
             - name: PHP-CS-Fixer
-              uses: docker://oskarstark/php-cs-fixer-ga:2.19.0
-              with:
-                args: --config=.php_cs.dist --allow-risky yes
+              uses: docker://oskarstark/php-cs-fixer-ga:latest
```
Comment (fashxp) [inline .github/workflows/php-cs-fixer.yaml]: should we really go for the latest, or tag at least for a major version?

**Review by fashxp — CHANGES_REQUESTED** [review]

File: .github/workflows/php-cs-fixer.yaml (hunk 1 above)
Comment (blankse) [reply .github/workflows/php-cs-fixer.yaml]: See https://github.com/pimcore/pimcore/blob/11.x/.github/workflows/php-cs-fixer.yaml and other repros

Comment (blankse) [reply .github/workflows/php-cs-fixer.yaml]: But yes I think we should set the latest major version. I will change it.

Comment (blankse) [reply .github/workflows/php-cs-fixer.yaml]: @fashxp Done https://github.com/pimcore/data-hub/pull/854/commits/759bb046266ce4c2f399156cffe86d4e5fef0e11

**Review by fashxp — APPROVED** [review]

**Comment by fashxp** [comment]
@blankse hmm, it seems that this image does not exist? see https://github.com/pimcore/data-hub/actions/runs/8797410558

**Comment by blankse** [comment]
@fashxp Maybe ":3" not work? Should we try it with ":latest"?

**Pull request merged**